main.go

```diff
@@ -2,35 +2,73 @@ package main
 
 import (
 	"encoding/json"
+	"html/template"
 	"log"
 	"net/http"
 	"os"
+	"strings"
 
 	"github.com/Clever/mesos-visualizer/ecs"
 )
 
 var (
-	Cluster            string
+	Clusters           map[string]string
 	AWSAccessKeyID     string
 	AWSSecretAccessKey string
 )
 
 func init() {
-	Cluster = getEnv("CLUSTER")
+	Clusters = getEnvJSON("CLUSTERS")
 	AWSAccessKeyID = getEnv("AWS_ACCESS_KEY_ID")
 	AWSSecretAccessKey = getEnv("AWS_SECRET_ACCESS_KEY")
 
 }
 
 func main() {
-	http.HandleFunc("/resources.json", resourcesHandler)
-	http.Handle("/", http.StripPrefix("/", http.FileServer(http.Dir("./static"))))
+	http.HandleFunc("/resources/", resourcesHandler)
+	http.HandleFunc("/", indexHandler)
+	http.Handle("/static/", http.StripPrefix("/static/", http.FileServer(http.Dir("./static"))))
 	log.Print("Listening on port 80...")
 	log.Fatal(http.ListenAndServe(":80", nil))
 }
 
-func resourcesHandler(w http.ResponseWriter, r *http.Request) {
-	c := ecs.NewClient(Cluster, AWSAccessKeyID, AWSSecretAccessKey)
+func indexHandler(w http.ResponseWriter, r *http.Request) {
+	tmpl, err := template.New("home").Parse(`
+	<!DOCTYPE html>
+	<head>
+		<meta charset="utf-8">
+		<title>ECS Visualizations</title>
+	</head>
+	<body>
+		{{ range $name, $arn := . }}
+		<h2>{{$name}}</h2>
+		<ul>
+			<li><a href="./static/sunburst.html?{{$name}}">Resource Utilization - Sunburst</a></li>
+			<li><a href="./static/treemap.html?{{$name}}">Resource Utilization - Treemap</a></li>
+		</ul>
+		{{end}}
+	</body>
+	`)
+	if err != nil {
+		panic(err)
+	}
+
+	err = tmpl.Execute(w, Clusters)
+	if err != nil {
+		panic(err)
+	}
+}
+
+func resourcesHandler(w http.ResponseWriter, req *http.Request) {
+	cluster := strings.TrimPrefix(req.URL.Path, "/resources/")
+	arn, ok := Clusters[cluster]
+	if !ok {
+		w.WriteHeader(404)
+		w.Write([]byte(`{"error": "unknow cluster"}`))
```

Comment:
*unknown

Reply:
Fixed in f78cc32e32358b00d1edd666a510a79d6a1f2f8b